test/extensions/filters/common/ext_authz/ext_authz_http_impl_test.cc

```diff
@@ -46,9 +46,8 @@ class ExtAuthzHttpClientTest : public testing::Test {
         .WillByDefault(ReturnRef(async_client_));
   }
 
-  static ClientConfigSharedPtr createConfig(const std::string& yaml = EMPTY_STRING,
-                                            uint32_t timeout = 250,
-                                            const std::string& path_prefix = "/bar") {
+  ClientConfigSharedPtr createConfig(const std::string& yaml = EMPTY_STRING, uint32_t timeout = 250,
```

Comment:
Do you have a test that covers the runtime override case?

Reply:
I have added a test for this.